snap-server.cabal

```diff
@@ -91,7 +91,7 @@ Library
     bytestring                          >= 0.9.1    && < 0.11,
     bytestring-builder                  >= 0.10.4   && < 0.11,
     case-insensitive                    >= 1.1      && < 1.3,
-    clock                               >= 0.4.1    && < 0.6,
+    clock                               >= 0.4.1    && < 0.7,
```

Comment:
@k-bx Oh interesting :P Thanks for the info! I'd be curious to know what other libs depend on clock and experience problems.